docs/source/commands/dev.mdx

```diff
@@ -1,88 +1,108 @@
+**To use `rover dev`, you need at least one running GraphQL API (subgraph).** Rover can obtain a subgraph's schema via introspection (either standard or [federated introspection](https://www.apollographql.com/docs/federation/subgraph-spec#enhanced-introspection-with-query_service)), or you can provide a local schema file.
 
-Here's an example `rover dev` command pointing to a locally running subgraph and using a file for the schema:
+Here's an example `rover dev` command that points to a locally running subgraph and provides its schema via a local file:
 
-```bash
+```bash title="Example command"
 rover dev --name products --schema ./products.graphql --url http://localhost:4000
 ```
 
-> Note that this doesn't _have_ to be a local subgraph, you can point to any URL!
-
-When starting a new `rover dev` session, Rover:
-
-1. Loads the subgraph schema
-2. Composes a [supergraph schema](/federation/federated-types/overview/#supergraph-schema)
-3. Starts a local [router](/router) so you can query your supergraph
-4. Watches the subgraph schema for changes (either via introspection or file) and recomposes the supergraph schema when it detects a change, auto-reloading the router
+> **You don't have to provide a _locally running_ subgraph!** You can point to _any_ GraphQL endpoint that your local environment can reach. You can even mix and match local and remote subgraphs, which is helpful for testing local changes against your staging subgraphs.
 
-## Connecting more subgraphs
+**When you start a `rover dev` session:**
 
-To add a new GraphQL API (i.e., subgraph) to a session, run `rover dev` in a new terminal with the necessary arguments for that new subgraph. If a session is already running, Rover automatically attaches subsequent `rover dev` processes to that existing session. For example, if you already have a session running with a subgraph named `products` (like in the above example), you can add a new subgraph named `reviews` with the following command:
+1. Rover obtains the subgraph schema you provide (via either introspection or file path).
+2. Rover composes a [supergraph schema](/federation/federated-types/overview/#supergraph-schema) from the subgraph schema.
+3. Rover starts a locally running [router](/router) instance and provides it the supergraph schema.
+4. Rover starts watching the provided subgraph schema for changes (via either introspection or file), and it recomposes the supergraph schema whenever it detects a change. This automatically reloads the router.
 
-```bash
-rover dev --name reviews --url http://localhost:4001
-```
+After you start a session, you can then [add other subgraphs to it.](#adding-a-subgraph-to-a-session)
 
-When you add a new subgraph to a session, Rover recomposes the supergraph schema and updates the router, so you can start querying multiple subgraphs through the same, still-running, endpoint.
+### Starting a session with multiple subgraphs
```

Comment:
Because this is unique to _starting_ the session, I made it a subitem of of the Starting section